lib/ScriptFileStorage.js

```diff
@@ -149,17 +150,20 @@ $class.listScripts = function(rootFolder, pattern, callback) {
   //    callback
   // );
 
+  debug('glob %s on %s', pattern, rootFolder);
+
   glob(
     pattern,
     {
       cwd: rootFolder,
+      follow: true,
+      realpath: true,
       strict: false
     },
     function(err, result) {
       if (result) {
-        result = result.map(function(relativeUnixPath) {
-          var relativePath = relativeUnixPath.split('/').join(path.sep);
-          return path.join(rootFolder, relativePath);
+        result = result.map(function(unixPath) {
+          return unixPath.split('/').join(path.sep);
```

Comment:
Are you sure the paths are returned as absolute, even though you specify rootFolder via `options.cwd` and not as `options.root`?

Related: 1c06f8ae6 and https://github.com/isaacs/node-glob/commit/f3105a3cfc12d3c6e0a2c931d8cf7f1a6a1ff49a